drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -398,6 +400,15 @@ def self.open_hoh_enrollments_for_project(project)
       end
     end
 
+    def self.external_form_types_for_project(project)
+      return [] unless project.present?
+
+      Hmis::Form::Instance.for_project(project).
+        joins(:definition).
```

Comment:
Small thing, I think you want preload here instead of joins. You'd want to use scope.preload() to avoid n+1 queries when referencing instance.definition in to_picklist_option
```suggestion
        preload(:definition).
```

Reply:
Thank you!